### PR TITLE
Fixes for nullable artwork images

### PR DESCRIPTION
**Pull request by dzucconi**
Fixes bug wherein images may be null.

### CODE REVIEW

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx [hunk 1]
```diff
@@ -30,6 +30,8 @@ export const DefaultHeaderArtwork: React.FC<DefaultHeaderArtworkProps> = ({
 }) => {
   const { trackEvent } = useTracking()
 
+  if (!artwork.node.image) return null
```
Comment (sweir27) [inline src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Curious what the UI treatment of this case is?

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx [hunk 2]
```diff
@@ -36,6 +36,8 @@ export const fitHeaderArtworks = (
         return headerArtworks
       }
 
+      if (!artwork.node.image) continue
```
Comment (damassi) [inline src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx:39]: 🤔This is pretty foreign syntax for most folks. Can it be expressed more conventionally?

Comment (dzucconi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx:39]: Lol, I can convert this while to a trampoline or recursive function or something. I also literally never use loops, but this is what was there.

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Just skips it.

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx (hunk 2 above)
Comment (damassi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx:39]: Ahh, i see, i didn't realized we were within a loop 👍 All good.

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx (hunk 1 above)
Comment (sweir27) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Yeah but does it mean there's no header image at all? (I'm not super familiar with how this works-- mind attaching a screenshot?)

Comment (dzucconi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Oh do you mean if the entire rail has no images? — I actually didn't test for that, will get back to this in a sec.

Comment (sweir27) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Ohh I see, this is just one image of many! This change seems 👍 in any case.

Comment (dzucconi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:33]: Yeah, I the naming of all these components leaves a lot to be desired.

**Comment by sweir27** [comment]
Good from me! Feel free to merge whenever.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.30.8`](https://github.com/artsy/reaction/releases/tag/v26.30.8) :rocket:

**Pull request merged**